Neos.TimeableNodeVisibility/Classes/Domain/HandlingResult.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Neos\TimeableNodeVisibility\Domain;
+
+use Neos\ContentRepository\Core\Projection\ContentGraph\Node;
+
+final class HandlingResult
+{
+    public const RESULT_ENABLED = 'ENABLED';
+    public const RESULT_DISABLED = 'DISABLED';
```

Comment:
do we want to make this state just a bool? It's internal anyways....

Reply:
Hmm, the property `$result` is public and used for console output. Also I'm struggling with naming of a bool, that can contain two result states: `$resultEnabled = false` does not mean that it acually was disabled. In fact yes, but the naming does not reflect that IMHO.

Reply:
just a thought, no need to mull over this too long :)

Reply:
im in a meeting will refactor this to be a enum :P

Reply:
Yes, no... this is to much...